<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Browser</class>
 <widget class="QDialog" name="Browser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="_tankButton">
       <property name="toolTip">
        <string>Tank</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/images/tank.png</normaloff>:/images/tank.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="_fileButton">
       <property name="toolTip">
        <string>Filesystem</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="_shotgunButton">
       <property name="toolTip">
        <string>Shotgun</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/images/shotgun.png</normaloff>:/images/shotgun.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>588</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="LineEdit" name="_searchBox">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QStackedWidget" name="_filterSwitcher">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="_tankFilterPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="TankFilterView" name="_tankFilterView" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="_fileFilterPage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="FileFilterView" name="_fileFilterView" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="_shotgunFilterPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="ShotgunFilterView" name="_shotgunFilterView" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="_revisionFilter" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="_resultsSwitcher">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="_revisionsPage1">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="_filterLabel">
                  <property name="text">
                   <string>Filter:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="_filterAll">
                  <property name="text">
                   <string>All</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="_filterLatest">
                  <property name="text">
                   <string>Latest</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="_filterRecommended">
                  <property name="text">
                   <string>Recommended</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>523</width>
                    <height>28</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="RevisionResultsView" name="_tankRevisionsView">
                <property name="rootIsDecorated">
                 <bool>false</bool>
                </property>
                <property name="uniformRowHeights">
                 <bool>true</bool>
                </property>
                <property name="sortingEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="_containersPage">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="ContainerResultsView" name="_tankContainersView"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="_hintLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>728</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="_buttons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TankFilterView</class>
   <extends>QWidget</extends>
   <header>.tank_filter</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RevisionResultsView</class>
   <extends>QTreeView</extends>
   <header>.revision_results</header>
  </customwidget>
  <customwidget>
   <class>FileFilterView</class>
   <extends>QWidget</extends>
   <header>.file_filter</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ShotgunFilterView</class>
   <extends>QWidget</extends>
   <header>.shotgun_filter</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ContainerResultsView</class>
   <extends>QTreeView</extends>
   <header>.container_results</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>..lineedit</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>_buttons</sender>
   <signal>accepted()</signal>
   <receiver>Browser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>970</x>
     <y>853</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_buttons</sender>
   <signal>rejected()</signal>
   <receiver>Browser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>970</x>
     <y>853</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_buttons</sender>
   <signal>accepted()</signal>
   <receiver>Browser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>921</x>
     <y>845</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>842</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
